### ISSUE TITLE
Parser of images do not accept url escaped string, contrary to what happens in the web version

### ISSUE THREAD

**Issue by tobiasBora** [mattermost/mattermost-mobile#5039]
#### Summary
It is impossible to link to an image that uses some urlescape codes, like in `![alpha](https://latex.codecogs.com/gif.latex?%5Calpha)`: I would expect to see this image: ![y](https://latex.codecogs.com/gif.latex?%5Calpha) but instead, nothing appears. However, if you remove the `%5C` char, then it displays an image... but of course not the expected one since we removed the important char (`%5C` encodes `\` that also fails to be recognized in Android version).

One reason why I'm interested by such feature is that want to use inline LaTeX in Mattermost on Android (normal inline LaTeX is not working on both web & android as explained here, and not-inlined LaTeX is not working in android) .

#### Environment Information

- OS Version: Android 10
- Mattermost App Version: 1.37.0 (Build 30000334) 
- Mattermost Server Version: 5.28.0 (Build 5.28.1)

#### Steps to reproduce

In any room, create this message:
```
Hello ![alpha](https://latex.codecogs.com/gif.latex?%5Calpha) I like math.
```

#### Expected behavior
I would expect to see a picture containing an alpha char like ![alpha](https://latex.codecogs.com/gif.latex?%5Calpha).

#### Observed behavior
Nothing is displayed.

**Comment by amyblais** [comment]
Opened a ticket: https://mattermost.atlassian.net/browse/MM-31341.

**Comment by amyblais** [comment]
Closing, this was fixed in v1.42 release,
